_caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: blog-post
+title: The use of HCD principles in conducting a needs assessment
+authors:
+  - author: Mutsa Carole Chinyamakobvu
+    authorTitle: 
+    authorGitHubHandle: mutsacc
+    authorGitHubID: 26001770
+
+tags:
+  #- tag: Data Collection and Management
```

Comment:
Hi @mutsacc,
It looks like you accidentally commented out the tag. Please fix.

Reply:
Thanks for picking that up @Rabia2219 - fixed.